docker/docker-poetry-install.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# Made this script so that it was easier
+# to run poetry install from the Dockerfile
+# with conditional logic - maybe there is an easier way?
```

Comment:
You can use a `ARG` to set an environment variable, and then check that here.  It could end up like:

```shell
cmd=(poetry install --no-interaction --no-ansi)

if [[ "$RPI" == "true" ]]; then
  cmd+=(--with rpi)
fi

"${cmd[@]}"
```

Reply:
You can also do it this way.  The loop and declare thing is maybe overkill for this, but it doesn't hurt anything.  (The cmd+= trick is good to know about though.)